crates/re_log_types/src/time.rs

```diff
@@ -62,15 +73,30 @@ impl Time {
         self.nanos_since_epoch() % (24 * 60 * 60 * 1_000_000_000) == 0
     }
 
+    // TODO(paris): Is there a pattern in Rust where you put free functions in an anonymous namespace versus in the struct?
+    // TODO(paris): Should we #[inline] this? I'm guessing the suggestion is to benchmark with and without, though this perhaps seems a bit big for inlining.
```

Comment:
Formatting is unlikely to be time critical for us, so I wouldn't put #inline without reason :)

Reply:
Sounds good, left as-is.